api/XRLightProbe.json

```diff
@@ -98,6 +100,57 @@
       "probeSpace": {
         "__compat": {
           "spec_url": "https://immersive-web.github.io/lighting-estimation/#dom-xrlightprobe-probespace",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/XRLightProbe/probeSpace",
+          "support": {
+            "chrome": {
+              "version_added": "90"
+            },
```

Comment:
@toji Did `XRLightProbe` actually ship on desktop?

Reply:
Given Chrome compat isn't the main purpose of this PR (it is adding mdn_urls), I think review can continue without an answer to this question. cc @vinyldarkscratch who added `XRLightProbe` in https://github.com/mdn/browser-compat-data/pull/11428, though.

Reply:
Chrome has shipped `XRLightProbe` on desktop based upon testing in https://mdn-bcd-collector.appspot.com/tests/api/XRLightProbe, so I'd say we're good to go on this.  This is all LGTM, let's merge it!  👍